fili-generic-example/src/main/resources/applicationConfig.properties

```diff
@@ -17,9 +17,12 @@ bard__fili_port=9998
 # Use memory for the default dimension backing store
 bard__dimension_backend=memory
 
-# Data Cache
-bard__druid_cache_enabled = false
-bard__druid_cache_v2_enabled = false
+# Data Cache strategy, which is one of the following
+# 1. ETagOnly
+# 2. LocalSignatureOnly (cache V2)
+# 3. TtlOnly (cache V1)
+# 4. None (no caching)
+bard__query_response_caching_strategy = None
```

Comment:
`None` or `NoCache`? Need to fix all the comments on all the .properties files also.